service-workers/redirects/fragment.https.window.js

```diff
@@ -0,0 +1,35 @@
+// META: script=/html/canvas/resources/canvas-tests.js
+// META: script=../service-worker/resources/test-helpers.sub.js
```

Comment:
Why are these tests in `service-workers/redirects` instead of `service-workers/service-workrer`?  We have other redirect tests in `service-workers/service-worker` so it doesn't make sense for this to be split off.  I personally would prefer to have all the service worker tests together.

Reply:
I might not understand the file structure of the service workers tests. Should I just move the directory to be a subdirectory of that one?

Reply:
That would be better, yes.  We don't really use many subdirectories in there, though.  I think there is only one other case of subdirectories in service-workers.  I guess I would recommend just putting the test in service-workrers/service-worker directly without a subdirectory.

Reply:
Sounds good. Once you all reach some conclusions in the new issue I'd be happy to update these tests.